src/plugins/Log.ts

```diff
@@ -90,14 +95,34 @@ class Log {
         }
     }
 
-    reportLog = (logInfo: any) => {
+    reportLog(logInfo: any): void {
         const logData = new LogData();
         const activeSpan = this.tracer ? this.tracer.getActiveSpan() : undefined;
         const spanId = activeSpan ? activeSpan.spanContext.spanId : '';
         logData.initWithLogDataValues(this.logData, spanId, logInfo);
 
         this.logs.push(logData);
     }
+
+    shimConsole(): void {
```

Comment:
I think, we need shim console between invocation start and finish. Because otherwise, since there is no active transaction, the logs won't be linked to anywhere. I mean we need to shim at the start of the request and revert back to original at the end of the request

Reply:
Done.  Shimming is done at` beforeInvocation` and reverted back at `afterInvocation`